src/Provider/GitHub/GitHubPRService.ts

```diff
@@ -31,6 +31,12 @@ export class GitHubPRService implements IGitHubPRService {
       userAgent: USER_AGENT,
       timeZone: TIME_ZONE,
       baseUrl: GitHubPRService.getApiBase(repoUrlObj),
+      log: {
+        debug: (message: string) => console.log('debug', message),
+        error: (message: string) => console.log('error', message),
+        info: (message: string) => console.log('info', message),
+        warn: (message: string) => console.log('warn', message),
+      },
```

Comment:
Use winston for logging (with correct log levels) like the rest of the app please

Reply:
this is also for debugging
will remove it